pkg/ledger/error.go

```diff
@@ -156,3 +156,26 @@ func NewLockError(err error) *LockError {
 		Err: err,
 	}
 }
+
```

Comment:
This error seems never used by the core, so it should be on the api package.

Reply:
You're right, fixed :)